core-web/libs/template-builder/src/lib/components/template-builder/components/remove-confirm-dialog/remove-confirm-dialog.component.scss

```diff
@@ -0,0 +1,7 @@
+@use "variables" as *;
+
+::ng-deep {
+    .p-button.p-button-text .pi.pi-trash {
```

Comment:
Can't we add classes to this buttons? because if so, we can add the severity, right?